langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -37,6 +37,10 @@ type AnydataType anydata;
 
 # Returns the number of members of an array.
 #
+# ```
+# [1, 2, 3, 4].length() => 4
```

Comment:
Options being considered atm (including some proposed by others too)

1. `# [1, 2, 3, 4].length() => 4`
2. `# [1, 2, 3, 4].length() >> 4`
3. `# [1, 2, 3, 4].length() == 4`
4. `# [1, 2, 3, 4].length() // 4`

Reply:
I prefer the option 4 because I kind of have doubt whether we could use options for all the available cases.

Reply:
```suggestion
# [1, 2, 3, 4].length() ⇒ 4
```

Reply:
If we use option 4,
user can copy paste the code and use in both shell and in vscode(after wrap the code in main method).

Reply:
How about this option? This is will not break the code snippet ( we can copy and paste the code as it is)
```ballerina
[1, 2, 3, 4].length() // => 4
```

Reply:
I think, better we use a different example for this. Either some strings in a list, just to avoid the confusion.

Reply:
This was already updated - https://github.com/ballerina-platform/ballerina-lang/pull/38989/commits/c88f6f680f4652a909d513b21e57e9688b4a1ce2#diff-2c948d23bf7d2faa264a43f44ca32e7f64f06b4dfb220764247218037b2d10f4R41